File: dotnet/source/cmf.eventing/IEventBus.cs
using System;

namespace cmf.eventing
{
    /// <summary>
    /// A convenience interface that combines the <see cref="IEventProducer">IEventProducer</see> 
    /// and <see cref="IEventConsumer">IEventConsumer</see> interfaces together 
    /// because they are most typically implemented and referenced together.
    /// </summary>
    public interface IEventBus : IEventProducer, IEventConsumer, IDisposable
    {
        
    }
}

File: java/cmf-eventing-api/src/main/java/cmf/eventing/IEventBus.java
package cmf.eventing;

import cmf.bus.IDisposable;

/**
 * A convenience interface that combines the {@link IEventProducer} and {@link IEventConsumer} 
 * interfaces together because they are most typically implemented and referenced together.
 */
public interface IEventBus extends IEventProducer, IEventConsumer, IDisposable {

}
